tests/Controller/EnMarche/MembershipControllerTest.php

```diff
@@ -114,7 +114,7 @@ public function testCreateMembershipAccountForFrenchAdherentIsSuccessful()
         $this->client->request(Request::METHOD_GET, $activateAccountUrl);
 
         $this->assertResponseStatusCode(Response::HTTP_FOUND, $this->client->getResponse());
-        $this->assertClientIsRedirectedTo('/adhesion?from_activation=1', $this->client);
```

Comment:
This test should not changed. It's a registration with a regular user so we should keep the query string parameter

Reply:
This is not used now